tests/album-update.test.js

```diff
@@ -0,0 +1,59 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Update Albums', () => {
+  let artists;
+  let album;
+  beforeEach(async () => {
+    const responses = await Promise.all([
+      db.query(
+        'INSERT INTO Artists (id, name, genre) VALUES( $1, $2, $3) RETURNING *',
+        [1, 'Tame Impala', 'rock']
+      ),
+    ]);
+
+    artists = responses.map(({ rows }) => rows[0]);
+
+    const albums_responses = await Promise.all([
```

Comment:
variables should be in camelCase
```suggestion
    const albumsResponses = await Promise.all([
```